airbyte-webapp/src/components/YamlEditor/YamlEditor.tsx

```diff
@@ -23,7 +24,8 @@ export const YamlEditor: React.FC = () => {
       base: "vs-dark",
       inherit: true,
       rules: [
-        { token: "string", foreground: styles.tokenString },
+        // add logic to convert these two six character
+        { token: "string", foreground: "#ffffff" },
```

Comment:
The changes to this file should probably also be removed from this PR before merging - I'm planning to address this specific issue separately

Reply:
[done](https://github.com/airbytehq/airbyte/pull/18410/commits/2d91e182862013ef2668068d280608dd78b8e9a7)